src/components/elements/MapPolygonPanel/ChecklistInformation.tsx

```diff
@@ -0,0 +1,82 @@
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalWithMap from "@/components/extensive/Modal/ModalWithMap";
+import { useModalContext } from "@/context/modal.provider";
+
+import Button from "../Button/Button";
+import Text from "../Text/Text";
+
+const ChecklistInformation = () => {
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerRequestPolygonSupport = () => {
+    openModal(
+      <ModalWithMap
+        title="Request Support"
```

Comment:
Looks like this component need `t()` as well.

Reply:
done